src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -257,6 +258,36 @@ def create_graph_enrichment(
     return FileBlock(svg_name, chart.render())
 
 
+def get_default_values_format(combination: ResourceKey) -> ChartValuesFormat:
+    if combination[1] == ResourceChartItemType.Node:
+        return ChartValuesFormat.Percentage
+    elif combination[0] == ResourceChartResourceType.CPU:
+        return ChartValuesFormat.CPUUsage
+    elif combination[0] == ResourceChartResourceType.Memory:
+        return ChartValuesFormat.Bytes
+    return ChartValuesFormat.Plain
+
+
+def __get_override_chart(prometheus_params: PrometheusParams, combination: ResourceKey) -> Optional[ChartOptions]:
+    if not prometheus_params.prometheus_graphs_overrides:
+        return
+    combinations = [
+        override
+        for override in prometheus_params.prometheus_graphs_overrides
+        if ResourceChartResourceType[override.resource_type] == combination[0]
+        and ResourceChartItemType[override.item_type] == combination[1]
+    ]
+
+    if len(combinations) == 0:
+        return None
+    if not combinations[0].values_format:
+        values_format = get_default_values_format(combinations[0])
```

Comment:
passing wrong object to get_default_values_format
should be `get_default_values_format(combination)`
will return error
 ```
   values_format = get_default_values_format(combinations[0])
  File "/app/src/robusta/core/playbooks/prometheus_enrichment_utils.py", line 262, in get_default_values_format
    if combination[1] == ResourceChartItemType.Node:
TypeError: 'OverrideGraph' object is not subscriptable
```